tests/Feature/InstallTest.php

```diff
@@ -140,6 +142,7 @@ public function testInstall(): void
 		/**
 		 * We now should NOT be redirected.
 		 */
+		Configs::dropCache();
```

Comment:
Why this change suddenly? How is this related to the PR?

Reply:
`Configs` is maintaining a static cache inside to avoid querying the table at each access and making it somewhat of a singleton.

During the execution of phpunit, the Configs singleton is never "cleaned", this is similar to sessions that are kept alive through multiple tests.

As a result because the Configs does not have username / password values from the start of the execution and not refreshed, the install/migration run is broken in the tests.